docs/OV_Runtime_UG/supported_plugins/HDDL.md

```diff
@@ -8,6 +8,10 @@ The Inference Engine HDDL plugin was developed for inference with neural network
 
 To configure your Intel® Vision Accelerator Design With Intel® Movidius™ on supported operating systems, refer to the Steps for Intel® Vision Accelerator Design with Intel® Movidius™ VPUs section in the installation guides for [Linux](../../install_guides/installing-openvino-linux.md) or [Windows](../../install_guides/installing-openvino-windows.md).
 
+> **NOTE**: The HDDL and Myriad plugins may cause conflicts when used at the same time.
+> To ensure proper operation in such a case, the number of booted devices needs to be limited in the 'hddl_autoboot.config' file.
+> Otherwise, the HDDL plugin will boot all available myriadx devices.
```

Comment:
What are the myriadx devices? Intel® Movidius™ Myriad™ X VPU -based devices?

Reply:
It would have to be confirmed by devs, but I assume we are talking about two different device categories: one is the entire device (that would be Intel Movidius etc.) and the other is the computing unit or core within the whole product which is also a "device." In this case I understand it is about the second one. After Maksim:
"it is expected behavior of HDDL plugin to boot all available myriadx cores"